packages/navigator/src/browser/navigator-filter.ts

```diff
@@ -58,7 +63,22 @@ export class FileNavigatorFilter {
     }
 
     protected createFilterPredicate(exclusions: FileNavigatorFilter.Exclusions): FileNavigatorFilter.Predicate {
-        return new FileNavigatorFilterPredicate(exclusions);
+        return new FileNavigatorFilterPredicate(this.interceptExclusions(exclusions));
+    }
+
+    hiddenFilesVisible(): boolean {
+        return this.showHiddenFilesFlag;
+    }
+
+    toggleHiddenFiles(visible: boolean): void {
```

Comment:
`toggle` should not expect a `boolean` but it should flip its internal `boolean` state.

Reply:
Method signature updated.